src/component/LogPages/LogPagesProps/LogPagesProps.tsx

```diff
@@ -0,0 +1,21 @@
+import "./LogPagesProps.scss";
+import React from 'react';
+
+export default function LogPagesProps({active, header, children, footer}) {
```

Comment:
a bit naming issue here, it seems this is an account page template component, not "props"